.github/workflows/publish-release-artifact.yml

```diff
@@ -0,0 +1,205 @@
+name: Publish artifact
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  determine-version:
+    runs-on: ubuntu-latest
+    outputs:
+      project-version: ${{ steps.set-version.outputs.project-version }}
+      is-lts: ${{ steps.set-version.outputs.is-lts }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Set version
+        id: set-version
+        run: |
+          version=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+
+          dot_count=$(echo $version | sed 's/[^.]//g' | awk '{ print length }')
+          if [ "${dot_count}" = '2' ]
+          then
+            is_lts=true
+          else
+            is_lts=false
+          fi
+
+          echo "Version is $version, is_lts: $is_lts"
+
+          echo "::set-output name=is-lts::${is_lts}"
+          echo "::set-output name=project-version::$version"
```

Comment:
Recommend moving this sort of thing to separate `*.sh` files for legibility. (You have already run `actions/checkout`.)

Reply:
this blocking?

Reply:
No, just a suggestion.